backend/composer/models.py

```diff
@@ -500,6 +500,15 @@ def npo_approved(self, *args, **kwargs):
     def exported(self, *args, **kwargs):
         ...
 
+    @transition(
+        field=state,
+        source=CSState.DRAFT,
+        target=CSState.EXPORTED,
+        permission=lambda instance, user: user.username == "system",
+    )
+    def exported_from_ingestion(self, *args, **kwargs):
+        ...
+
```

Comment:
@zsinnema please have a look at this addition. I'm not sure if it's the best way to do what I want which is to move the newly ingested statements that are on draft by default to exported

Reply:
@afonsobspinto I would not add this transition since it will become part of the "statement flow" process.

The way you solved it last time with the `.update` sound better to me. What do you think?

Reply:
I'm afraid my original strategy doesn't work because on updates we get an error 'Direct state modification is not allowed' thus the need for the new transition. But I do understand your concerns. Do you have any alternative solution @zsinnema ?